vue/components/SharedSubComponents/loadingSpinner.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <div class="page-load-status">
+        <div v-if="loading" class="loader-ellips infinite-scroll-request">
+            <span class="loader-ellips__dot"></span>
+            <span class="loader-ellips__dot"></span>
+            <span class="loader-ellips__dot"></span>
+            <span class="loader-ellips__dot"></span>
+        </div>
```

Comment:
How does this animate? It's not clear to me.

Reply:
No clue to be honest, that HTML has been floating around the project since before I joined it. I assume it's a CSS animation. I guess I might as well look into it at some point